xl2times/utils.py

```diff
@@ -315,6 +315,56 @@ def round_sig(x, sig_figs):
 default_log_name = "log" if default_log_name == "" else default_log_name
 
 
+def _is_match_on_windows(path: Path, pattern: str) -> bool:
+    """Hack for case-insensitive comparisson, because case_sensitive
+    parameter in match available from Python 3.12, but not in 3.11.
+    """
+    return PureWindowsPath(path).match(pattern)
+
+
+def is_veda_based(files: list[str]) -> bool:
+    """Determine whether the model follows Veda file structure.
+    This function does not verify file extension.
+    """
+    marker = "SysSettings.*"
+
+    matches = [file for file in files if _is_match_on_windows(Path(file), marker)]
```

Comment:
```suggestion
    matches = [file for file in files if str(file.stem).lower().startswith("syssettings.")]
```
Not tested, but I think that'll work

Reply:
I think it will! But about the other patterns that inluclude folders, etc?